sqldelight-gradle-plugin/build.gradle

```diff
@@ -64,4 +72,19 @@ test {
   }
 }
 
+tasks.getByName("shadowJar").configure {
+  archiveClassifier.set("")
+  configurations = [project.configurations.shade]
+  minimize()
+  relocate "com.intellij", "com.squareup.sqldelight.com.intellij"
+
+  exclude '*.properties'
+  exclude 'META-INF/services/*'
```

Comment:
theres even like shell scripts inside of the shadowJar lol...im surprised these arent all being cleaned up by minimize